### DISCUSSION TITLE
request information

### DISCUSSION THREAD

**Discussion by x-077** [strongdm/contrib#19] [General]
Hello, 

I've been working on a ansible collection to wrap your python API. I would like to know if there is anything in particular that I should know if I want to submit it to your repo ? 

Thanks

**Comment by camposer** [comment]
Hi @matth-c3 

Thanks for considering becoming one of our contributors, awesome!

Just fork the repo, create a folder -following the structure that we have-, add a nice README and create a PR 😀

**Reply by x-077** [reply]
Hello @camposer , 

ok thanks, I will do the necessary this week.
